HealthAPILibrary/GiniHealthAPILibrary/Sources/GiniHealthAPILibrary/Documents/ExtractionResult.swift

```diff
@@ -7,25 +7,6 @@
 
 import Foundation
 
-/**
```

Comment:
[Clarification]: where did we move it?

Reply:
Into GiniMerchantSDK and into GiniHealthSDK, because we parse different fields on both SDKs

Reply:
Look for `public enum ExtractionType: String {`